client/my-sites/importer/importer-logo.jsx

```diff
@@ -19,34 +19,37 @@ import SocialLogo from 'components/social-logo';
  */
 import './importer-logo.scss';
 
-const ImporterLogo = ( { icon } ) => {
+const ImporterLogo = ( { icon, className = 'importer__service-icon' } ) => {
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+	if ( 'blogger' === icon ) {
```

Comment:
Why this special handling for `blogger`?

Reply:
Instead of `icon={ importSiteEngine }`, we can get this mapping (that the icon for `blogger` is `blogger-alt`) from `client/lib/importer/importer-config`. It doesn't seem right to map engine to the actual icon configured for the engine within this component.

Reply:
Meeps, this was because we have two icons for Blogger, and use the version called `blogger-alt`. Thanks Jon, now changed.